From c3a004b8d6dff1a767573dd7baf5d45efddb9f2b Mon Sep 17 00:00:00 2001
From: Mingli Yu <mingli.yu@windriver.com>
Date: Fri, 15 Jul 2016 11:23:17 +0800
Subject: [PATCH] grubby: remove bashisms in test.sh

* Update #!/bin/bash to #!/bin/sh in test.sh to
  remove below warning:
  QA Issue: /usr/lib/grubby/ptest/test.sh contained
  in package grubby-ptest requires /bin/bash, but no
  providers found in RDEPENDS_grubby-ptest? [file-rdeps]

Signed-off-by: Mingli Yu <mingli.yu@windriver.com>
---
 test.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test.sh b/test.sh
index f738388..f3537f3 100755
--- a/test.sh
+++ b/test.sh
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 #
 # test.sh -- grubby regression tests
 #
-- 
2.8.1

